### DISCUSSION TITLE
Est-ce que l'on devrait améliorer notre authentification.

### DISCUSSION THREAD

**Discussion by HiroKX** [HiroKX/planifecation#187] [Polls]
L'authentification sur l'application utilise pour l'instant un JWT avec Refresh Token qui n'est pas parfaite car le refresh token est exposé régulièrement.

Il faudrait voir pour l'améliorer avec le rafraichissement du Refresh Token afin de faire un silent login parfait.

**Comment by HiroKX** [comment]
Je pense personnellement qu'il ne faudrait pas le faire pour l'instant car cela utiliserait du temps de développement qui est nécessaire au développement de feature plus évolué.

**Reply by Zeyel** [reply]
Oui la connexion est assez robuste bien que pas parfaite pour le temps de développement que l'on a.
